### DISCUSSION TITLE
Index.html QC report empty

### DISCUSSION THREAD

**Discussion by jb21469** [HALFpipe/HALFpipe#648] [Q&A]
Hello,

First, thank you so much for HALFpipe! It is such an amazing tool.

I've successfully run one subject through HALFpipe v1.2.2 on our HPC. I went to QC their data but when I opened the index.html file in the browser no content appeared. It seems that all the necessary figures and js files were created. However, in the err.txt file I did get the following warning:

`[2023-11-11 00:33:11,0996] [py.warnings     ] [WARNING  ] /usr/local/miniconda/lib/python3.10/site-packages/nilearn/plotting/displays/_axes.py:71: UserWarning: No contour levels were found within the data range.
\u2502   im = getattr(ax, type)(data_2d.copy(),
\u2502 Stack (most recent call last):
\u2502   File "<string>", line 1, in <module>
\u2502   File "/usr/local/miniconda/lib/python3.10/multiprocessing/forkserver.py", line 274, in main
\u2502     code = _serve_one(child_r, fds,
\u2502   File "/usr/local/miniconda/lib/python3.10/multiprocessing/forkserver.py", line 313, in _serve_one
\u2502     code = spawn._main(child_r, parent_sentinel)
\u2502   File "/usr/local/miniconda/lib/python3.10/multiprocessing/spawn.py", line 129, in _main
\u2502     return self._bootstrap(parent_sentinel)
\u2502   File "/usr/local/miniconda/lib/python3.10/multiprocessing/process.py", line 315, in _bootstrap
\u2502     self.run()
\u2502   File "/usr/local/miniconda/lib/python3.10/multiprocessing/process.py", line 108, in run
\u2502     self._target(*self._args, **self._kwargs)
\u2502   File "/usr/local/miniconda/lib/python3.10/concurrent/futures/process.py", line 243, in _process_worker
\u2502     r = call_item.fn(*call_item.args, **call_item.kwargs)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/halfpipe/plugins/multiproc.py", line 72, in run_node
\u2502     result["result"] = node.run(updatehash=updatehash)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/nipype/pipeline/engine/nodes.py", line 524, in run
\u2502     result = self._run_interface(execute=True)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/nipype/pipeline/engine/nodes.py", line 642, in _run_interface
\u2502     return self._run_command(execute)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/nipype/pipeline/engine/nodes.py", line 719, in _run_command
\u2502     result = self._interface.run(cwd=outdir, ignore_exception=True)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/nipype/interfaces/base/core.py", line 399, in run
\u2502     runtime = self._post_run_hook(runtime)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/nipype/interfaces/mixins/reporting.py", line 50, in _post_run_hook
\u2502     self._generate_report()
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/niworkflows/interfaces/masks.py", line 340, in _generate_report
\u2502     plot_segs(
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/niworkflows/viz/utils.py", line 252, in plot_segs
\u2502     svg = _plot_anat_with_contours(
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/niworkflows/viz/utils.py", line 288, in _plot_anat_with_contours
\u2502     display.add_contours(segs[i], levels=levels[i], **plot_params)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/nilearn/plotting/displays/_slicers.py", line 315, in add_contours
\u2502     self._map_show(img, type='contour', threshold=threshold, **kwargs)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/nilearn/plotting/displays/_slicers.py", line 403, in _map_show
\u2502     im = display_ax.draw_2d(data_2d, data_bounds, bounding_box,
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/nilearn/plotting/displays/_axes.py", line 71, in draw_2d
\u2502     im = getattr(ax, type)(data_2d.copy(),
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/matplotlib/__init__.py", line 1412, in inner
\u2502     return func(ax, *map(sanitize_sequence, args), **kwargs)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/matplotlib/axes/_axes.py", line 6301, in contour
\u2502     contours = mcontour.QuadContourSet(self, *args, **kwargs)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/matplotlib/contour.py", line 812, in __init__
\u2502     kwargs = self._process_args(*args, **kwargs)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/matplotlib/contour.py", line 1441, in _process_args
\u2502     x, y, z = self._contour_args(args, kwargs)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/matplotlib/contour.py", line 1492, in _contour_args
\u2502     self._process_contour_level_args(args)
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/matplotlib/contour.py", line 1187, in _process_contour_level_args
\u2502     _api.warn_external(
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/matplotlib/_api/__init__.py", line 299, in warn_external
\u2502     warnings.warn(message, category, stacklevel)
\u2502   File "/usr/local/miniconda/lib/python3.10/warnings.py", line 109, in _showwarnmsg
\u2502     sw(msg.message, msg.category, msg.filename, msg.lineno,
\u2502   File "/usr/local/miniconda/lib/python3.10/site-packages/halfpipe/logging/base.py", line 35, in showwarning
\u2514\u2500    logger.warning(f"{s}", stack_info=True)`

Do you have any suggestions as to what I can do to be able to view the QC report?

Thank you in advance,
Jenna

**Comment by HippocampusGirl** [comment]
Hi Jenna,

I'm sorry you're having these issues! Let's see if we can figure this out.
Were there any files or folders in the `reports` folder besides the `index.html` file?

Best,
Lea
